src/pages/Acknowledgements/pageContent.tsx

```diff
@@ -1,193 +1,289 @@
+          <FormattedMessage defaultMessage="More on Eth2 economics" />
         </Link>
       </>
     ),
-    acknowledgementText:
-      'I understand that it is important to keep my validator online and updated.',
+    acknowledgementText: (
+      <FormattedMessage defaultMessage="I understand that it is important to keep my validator online and updated." />
+    ),
   },
   [AcknowledgementIdsEnum.slashing]: {
-    title: 'Slashing Risks',
+    title: <FormattedMessage defaultMessage="Bad validator behaviour" />,
     content: (
       <>
-        <Text size="large" className="my10">
-          Validators that act maliciously, or contrary to the specification, are
-          liable to be slashed (incur a large penalty).
+        <Text size="medium" className="my10">
+          <FormattedMessage
+            defaultMessage="If you try to cheat the system, or act contrary to the specification,
+              you will be liable to incur a medium penalty, known as slashing."
+          />
         </Text>
-        <Link to="/faq" className="my10" primary withArrow>
-          Learn More about about slashing
+        <Link
+          to="https://github.com/ethereum/eth2.0-specs"
+          className="my10"
+          primary
+        >
+          <FormattedMessage defaultMessage="The Eth2 specification" />
+        </Link>
+        <Link to="/faq" className="my10" primary>
+          <FormattedMessage defaultMessage="More on slashing risks" />
         </Link>
       </>
     ),
-    acknowledgementText:
-      'I understand that if I act contrary to the specification, I am liable to be slashed.',
+    acknowledgementText: (
+      <FormattedMessage
+        defaultMessage="I understand that if I act contrary to the specification,
+          I am liable to be slashed."
+      />
+    ),
   },
   [AcknowledgementIdsEnum.keyManagement]: {
-    title: 'Backup Mnemonic',
+    title: <FormattedMessage defaultMessage="Key management" />,
     content: (
-      <Text size="large" className="my10">
-        Validator keys are derived from a unique mnemonic (or seed). Your seed
-        is the ONLY WAY to withdraw your funds. Above all, keep it safe!
-      </Text>
+      <>
+        <Text size="medium" className="mt10">
+          <FormattedMessage
+            defaultMessage="To become a validator you'll need to know about managing keys and
+              protecting a mnemonic. If you are not yet familiar with keys and mnemomics, please
+              do not proceed."
+          />
+        </Text>
+        <Text size="medium" className="mt20">
+          <FormattedMessage defaultMessage="We'll help you create a signing key and a withdrawal key for every validator you want to run. Signing and withdrawal keys are separate by design. Your signing key must be held online because it needs to be available at all times. Your withdrawal key doesn't need to be online so it's better to keep it separate for security." />
```

Comment:
I worry as it currently stands, this implies that a user will get their withdrawal keys now.
```suggestion
          <FormattedMessage defaultMessage="We'll help you create a signing key for every validator you want to run. Because there are no withdrawals until The Merge, you will not create your withdrawal keys now. When it is possible to withdraw you funds, you can derive your withdrawal keys from your mnemonic." />
```